tests/xcconfig/BUILD.bazel

```diff
@@ -39,4 +39,12 @@ apple_framework(
     },
 )
 
+apple_framework(
+    name = "wont_build_without_xcconfig",
+    non_arc_srcs = ["weak_property.m"],
```

Comment:
Do we know for sure that this option is working properly? Could potentially explicitly add a call to `addRef` and `release` so that compiling the code double checks that.

Reply:
We do, note that this target fails to compile without the xcconfig applied